examples/code/home/configuration/engine/config_template.yaml

```diff
@@ -1,56 +1,14 @@
-telemetry:
-  environment: ${env:TELEMETRY_ENVIRONMENT}
-  version: ${env:VERSION}
-  logging:
-    level: ${env:TELEMETRY_LOGGING_LEVEL} # debug, info, warn, error
-    encoding: ${env:TELEMETRY_LOGGING_ENCODING}
+# yaml-language-server: $schema=https://raw.githubusercontent.com/ArcadeAI/schemas/main/engine/config/1.0/schema.json
```

Comment:
Included this line because it lights up VSCode's intellisense for YAML and I want people to have that if they copy this config template.

It is annoying that the `$schema:` line doesn't do this, there is not yet an official way to declare a schema for YAML. 😛  https://github.com/yaml/yaml-spec/issues/33